vendor/vendor.json

```diff
@@ -261,776 +261,808 @@
+			"revisionTime": "2018-04-04T18:37:54Z",
+			"version": "v1.13.28",
+			"versionExact": "v1.13.28"
 		},
 		{
-			"checksumSHA1": "lAgaKbwpyflY7+t4V3EeH18RwgA=",
+			"checksumSHA1": "nlvFlqVKHyuyYHG88H3Yt/OA1e0=",
 			"path": "github.com/aws/aws-sdk-go/service/lambda",
-			"revision": "ebc8a649dc2a7c78c9237c2575a63b3efd6d0a2d",
-			"revisionTime": "2018-02-12T21:53:52Z",
-			"version": "v1.12.75",
-			"versionExact": "v1.12.75"
+			"revision": "7f68df2a5baf19398d17def23a514a6e617e5937",
+			"revisionTime": "2018-04-04T18:37:54Z",
+			"version": "v1.13.28",
+			"versionExact": "v1.13.28"
 		},
 		{
-			"checksumSHA1": "4yGrGQatXcr8eGRWUoBg3KqAHK8=",
+			"checksumSHA1": "AO97b1GChOs2geiFc//6YcpVfX8=",
+			"path": "github.com/aws/aws-sdk-go/service/lexmodelbuildingservice",
+			"revision": "f63b41ba024702aca32b063327a0852f9758d914",
+			"revisionTime": "2018-04-19T23:04:09Z"
+		},
+		{
+			"checksumSHA1": "jyxQ6TEdSl9/j7qC0LCkVMSXkog=",
 			"path": "github.com/aws/aws-sdk-go/service/lightsail",
-			"revision": "ebc8a649dc2a7c78c9237c2575a63b3efd6d0a2d",
-			"revisionTime": "2018-02-12T21:53:52Z",
-			"version": "v1.12.75",
-			"versionExact": "v1.12.75"
+			"revision": "7f68df2a5baf19398d17def23a514a6e617e5937",
+			"revisionTime": "2018-04-04T18:37:54Z",
+			"version": "v1.13.28",
+			"versionExact": "v1.13.28"
 		},
 		{
-			"checksumSHA1": "kEZmPI9Y9+05SWuRCdtt+QkqwLI=",
+			"checksumSHA1": "I1sxeu8aJOiM3a5/tnJya8blE3A=",
 			"path": "github.com/aws/aws-sdk-go/service/mediastore",
-			"revision": "ebc8a649dc2a7c78c9237c2575a63b3efd6d0a2d",
-			"revisionTime": "2018-02-12T21:53:52Z",
-			"version": "v1.12.75",
-			"versionExact": "v1.12.75"
+			"revision": "7f68df2a5baf19398d17def23a514a6e617e5937",
+			"revisionTime": "2018-04-04T18:37:54Z",
+			"version": "v1.13.28",
+			"versionExact": "v1.13.28"
 		},
 		{
-			"checksumSHA1": "3QV+ZVkQ8g8JkNkftwHaOCevyqM=",
+			"checksumSHA1": "lAweYYdjcZ6acj6oI3HI9hX3eIQ=",
 			"path": "github.com/aws/aws-sdk-go/service/mq",
-			"revision": "ebc8a649dc2a7c78c9237c2575a63b3efd6d0a2d",
-			"revisionTime": "2018-02-12T21:53:52Z",
-			"version": "v1.12.75",
-			"versionExact": "v1.12.75"
+			"revision": "7f68df2a5baf19398d17def23a514a6e617e5937",
+			"revisionTime": "2018-04-04T18:37:54Z",
+			"version": "v1.13.28",
+			"versionExact": "v1.13.28"
 		},
 		{
-			"checksumSHA1": "QuOSKqV8nFvvzN4wcsToltMFI1Y=",
+			"checksumSHA1": "X/WdGuHGMxRCvufmQxstVjdbFT4=",
 			"path": "github.com/aws/aws-sdk-go/service/opsworks",
-			"revision": "ebc8a649dc2a7c78c9237c2575a63b3efd6d0a2d",
-			"revisionTime": "2018-02-12T21:53:52Z",
-			"version": "v1.12.75",
-			"versionExact": "v1.12.75"
+			"revision": "7f68df2a5baf19398d17def23a514a6e617e5937",
+			"revisionTime": "2018-04-04T18:37:54Z",
+			"version": "v1.13.28",
+			"versionExact": "v1.13.28"
+		},
+		{
+			"checksumSHA1": "smOJ/rypXb18/Rjryf2a85paSa4=",
+			"path": "github.com/aws/aws-sdk-go/service/organizations",
```

Comment:
Nitpick: These new dependencies were pulled in via git revision instead of a matching tag similar to the rest of the aws/aws-sdk-go dependencies.

```
diff --git a/vendor/vendor.json b/vendor/vendor.json
index f9ca8daff..90de5b930 100644
--- a/vendor/vendor.json
+++ b/vendor/vendor.json
@@ -903,8 +903,10 @@
 		{
 			"checksumSHA1": "AO97b1GChOs2geiFc//6YcpVfX8=",
 			"path": "github.com/aws/aws-sdk-go/service/lexmodelbuildingservice",
-			"revision": "f63b41ba024702aca32b063327a0852f9758d914",
-			"revisionTime": "2018-04-19T23:04:09Z"
+			"revision": "7f68df2a5baf19398d17def23a514a6e617e5937",
+			"revisionTime": "2018-04-04T18:37:54Z",
+			"version": "v1.13.28",
+			"versionExact": "v1.13.28"
 		},
 		{
 			"checksumSHA1": "jyxQ6TEdSl9/j7qC0LCkVMSXkog=",
@@ -941,8 +943,10 @@
 		{
 			"checksumSHA1": "smOJ/rypXb18/Rjryf2a85paSa4=",
 			"path": "github.com/aws/aws-sdk-go/service/organizations",
-			"revision": "f63b41ba024702aca32b063327a0852f9758d914",
-			"revisionTime": "2018-04-19T23:04:09Z"
+			"revision": "7f68df2a5baf19398d17def23a514a6e617e5937",
+			"revisionTime": "2018-04-04T18:37:54Z",
+			"version": "v1.13.28",
+			"versionExact": "v1.13.28"
 		},
 		{
 			"checksumSHA1": "vL235vodhgG61BkUxRD4B7io+zk=",
```